packages/linearprogress/src/css/index.js

```diff
@@ -13,23 +16,23 @@ export default {
     borderRadius: '4px'
   },
   [`.psds-linearprogress__bg.psds-theme--${themeDefaultName}`]: {
-    backgroundColor: transparentize(0.8, core.colors.bone)
+    backgroundColor: 'rgba(255, 255, 255, 0.15)'
```

Comment:
🤷‍♂ - no utility usage eh?

Reply:
It's not background utility.  And it's not a border.  Brian and I talked about semantics and came to the conclusion of using a one-off.